docs/upgrade/troubleshooting.md

```diff
@@ -200,3 +200,14 @@ deployment.apps/hvst-upgrade-xxxxx-upgradelog-downloader scaled
 ```
 
 :::
+
+### Cleanup unused images for enough disks to avoid image garbage collections
+
+The default value of `imageGCHighThresholdPercent` in [KubeletConfiguration](https://kubernetes.io/docs/reference/config-api/kubelet-config.v1beta1/#kubelet-config-k8s-io-v1beta1-KubeletConfiguration) is `85`. If kubelet detects disk usage is more than 85%, it tries to remove unused images.
+
+During Harvester upgrade, the system loads new images to each node. If disk usage is more than 85%, new images may be candidates for cleanup, because it's not used by any container.
```

Comment:
`If disk usage exceeds 85%, these new images may be marked for cleanup since they are not used by any containers.`